modules/web-console/frontend/app/configuration/generator/generator/ConfigurationGenerator.js

```diff
@@ -2309,7 +2309,7 @@ export default class IgniteConfigurationGenerator {
         if (available('2.0.0'))
             ccfg.enumProperty('partitionLossPolicy');
 
-        ccfg.intProperty('copyOnRead');
+        ccfg.boolProperty('copyOnRead');
```

Comment:
@vsisko is this the fix for "BigDecimal" issue?

Reply:
It is a typo that does not has influence on work of Web Console.
It was noticed by @akuznetsov-gridgain in the previous comments.